cypress/integration/1-getting-started/todo.spec.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
+
```

Comment:
Cypress scaffolds out many examples of test writing in the integration folder so you can follow along in how to arrange/act/asset